java/src/server/models/ServerModel.java

```diff
@@ -1,40 +1,64 @@
 package server.models;
 
+import org.jetbrains.annotations.NotNull;
+
+import java.util.HashMap;
 import java.util.Map;
 
 public class ServerModel {
 
-    private Map<Integer, GameModel> gameIdToModel;  //game id to game model
-    private Map<Integer, User> idToUser;  //id to users
+    private Map<Integer, User> idToUser;
+    private Map<Integer, GameModel> gameIDToModel;
 
     public ServerModel() {
+        idToUser = new HashMap<>();
+        gameIDToModel = new HashMap<>();
     }
 
     /**
-     * registers a User
+     * Called when a new {@link User} registers.  This method inserts a new {@link User} into {@link ServerModel#idToUser}
+     *
+     * @pre The Register button is clicked
+     * the credentials ({@link User#username} and {@link User#password}) are valid
+     * @post A new {@link User} is placed in {@link ServerModel#idToUser}
      */
     public void registerUser() {
 
     }
 
     /**
-     * gets the User that corresponds to the id parameter
+     * Called when a new game is created.  This method creates a new {@link GameModel} and inserts it into {@link ServerModel#gameIDToModel}
      *
-     * @param id a User's id
-     * @return the User that corresponds to the id parameter
+     * @pre The Create Game button is clicked before a game is started
+     * each {@link shared.models.game.Player} are valid with their game color
+     * the game name is valid
+     * @post A new {@link GameModel} is placed in {@link ServerModel#gameIDToModel}
```

Comment:
Documentation should not refer to internal constructs. Instead, it should say that `#getGameModel` should be valid for some id. In fact, that new ID should be returned in `startNewGame`